lib/server/services/environment/EnvironmentService.js

```diff
@@ -117,6 +117,52 @@ class EnvironmentService {
         return this.get(environmentId);
     }
 
+    /**
+     * Define environment state at a given time
+     *
+     * @param {string} environmentId the id of the environment for which a new state must be registered
+     * @param {EnvironmentState} state the state to save
+     * @param {object<string, string>} vars updated environment vars
+     * @return {Promise<void>} resolves once the state has been saved
```

Comment:
`vars` ?

Reply:
Named vars in ECS, but I agree with you, renamed